app/grandchallenge/container_exec/emails.py

```diff
@@ -22,17 +22,37 @@ def send_invalid_dockerfile_email(*, container_image):
         f"This is an automated service email from {site.domain}."
     )
 
-    recipients = list(get_user_model().objects.filter(is_staff=True))
+    staff_message = (
+        f"Dear {{}},\n\n"
```

Comment:
Using `{}` is ok with only 1 argument, but for 2 arguments to `format` could you use keywords? See "accessing arguments by name" here https://docs.python.org/3.8/library/string.html#format-examples